### ISSUE TITLE
Disk2VHD - command line argument for devices without drive letter missing

### ISSUE THREAD

**Issue by nona00** [MicrosoftDocs/sysinternals#652]
Disk2VHD 2.02 still misses a command line parameter to select partitions which only have an ID but no drive letter (for example the hidden EFI or recovery partitions)

![grafik](https://user-images.githubusercontent.com/3192606/211810260-2a56e51f-35b5-4656-b10a-8205703f6796.png)

**Comment by cap4cho** [comment]
News about this?

**Comment by cap4cho** [comment]
> @foxmsft: any news on this?

Yes, but no with disk2vhd, im using starwind v2v converter with command line arguments
